TelCo.ColorCoder/IndexHelper.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    class IndexHelper
+    {
+        internal static int getMajorIndex(ColorPair pair,Color[] colorMapMajor)
+        {
+            int majorIndex = -1;
+            for (int i = 0; i < colorMapMajor.Length; i++)
+            {
+                if (colorMapMajor[i] == pair.MajorColor)
+                {
+                    majorIndex = i;
+                    break;
+                }
+            }
+            return majorIndex;
+        }
+
+        internal static int getMinorIndex(ColorPair pair, Color[] colorMapMinor)
+        {
```

Comment:
This function is very similar to getMajorIndex. Can you think of a way to reduce duplication?

Reply:
Thanks sudeep for pointing out that duplication. I have merged the two functions into single function and hence removed the duplication.